snakewm/wm.py

```diff
@@ -148,7 +148,33 @@ def set_bg_image(self, file):
         if file_extension == ".jpg" or file_extension == ".png":
             self.BG = pygame.transform.scale(pygame.image.load(file), self.DIMS)
 
+    def load_task_bar(self):
+        menu_size_x = 100
+        menu_size_y = 50
+        menu_pos_x = 0
+        menu_pos_y = self.DIMS[1] - menu_size_y
+        self.menu_button = pygame_gui.elements.UIButton(
+            relative_rect=pygame.Rect(
+                (menu_pos_x, menu_pos_y), (menu_size_x, menu_size_y)
+            ),
+            text="Menu",
+            manager=self.MANAGER,
+        )
+
+    def check_main_menu_state(self):
```

Comment:
This method might be easier to understand if it were named `toggle_menu_state()`

Reply:
@callmepavo yeess! Let me push